projects/plugins/jetpack/extensions/blocks/blogroll/blogroll-item/editor.scss

```diff
@@ -1,41 +1,63 @@
-.wp-block-jetpack-blogroll-item {
-	display: flex;
-	padding: 20px 8px;
-	gap: 20px;
+.wp-block-jetpack-blogroll {
 
-	font-style: normal;
-	font-weight: 400;
-
-		figure {
-			margin: 0;
-			height: 48px;
-			width: 48px;
+	h3 + div .jetpack-blogroll-item-divider {
+		display: none;
+	}
 
-			&.empty-site-icon {
-				background: url('../placeholder-site-icon.svg');
-				background-size: cover;
+	.jetpack-blogroll-item-divider {
+		margin-block-start: 0;
+		margin-block-end: 0;
+		border-top: 1px solid var(--wp--preset--color--foreground);
+		border-bottom: 0;
+		border-right: 0;
+		border-left: 0;
+		opacity: 0.2;
+	}
 
-				.blogroll-item-image {
-					display: none;
+	.wp-block-jetpack-blogroll-item {
+		display: flex;
+		padding: 20px 0;
+		gap: 20px;
+	
+		font-style: normal;
+		font-weight: 400;
+	
+			figure {
+				margin: 0;
+				height: 48px;
+				width: 48px;
+				align-self: flex-start;
+	
+				&.empty-site-icon {
+					background: url('../placeholder-site-icon.svg');
+					background-size: cover;
+	
+					.blogroll-item-image {
+						display: none;
+					}
 				}
 			}
+	
+		.blogroll-item-image {
+			border-radius: 50%;
+			width: 48px;
+			height: 48px;
+			max-width: 48px;
 		}
+	
+		p.jetpack-blogroll-item-title.rich-text {
+			color: inherit;
+			font-weight: bold;
```

Comment:
Related to https://github.com/Automattic/jetpack/issues/37929 - Update title to match paragraph font-size and add bold styling:

This will force the title to be bold and will not allow users to customize it. I'm unsure how to set the title font weight to bold by default and allow customizations.

Reply:
cc: @crisbusquets - keeping you in the loop

Reply:
If we are forcing bold styles for the title then maybe we can remove the `core/bold` format from allowedFormats.

Reply:
Yeah, thats a good idea since the formatting won't work after forcing the bold style anyways.